src/backend/marsha/core/models/file.py

```diff
@@ -129,7 +129,7 @@ class Meta:
             )
         ]
 
-    def get_source_s3_key(self, stamp=None):
+    def get_source_s3_key(self, stamp=None, extension=None):
```

Comment:
not very confortable with the fact that document.extension is "jpg" and here extension is ".jpg"...
Names should convey the presence or not of the dot? Or maybe better pass it here without the dot?

Reply:
`splitext` and `guess_extension` return the extension with the leading dot, this is why I did that, I should remove it yes.

Reply:
I made all the work in the `Document.get_source_s3_key` method. The method accepts an extension with a leading dot or not and deal with it. It is less error prone I think.